ground/gcs/src/app/main.cpp

```diff
@@ -264,6 +265,9 @@ int main(int argc, char **argv)
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
     }
+    QProcess process;
```

Comment:
Thanks for finding this.  Argh.  I was hoping we could just edit info.plist -- it's pretty hacky to have to run a subprocess.